x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/inference_pipeline_card.tsx

```diff
@@ -51,151 +50,137 @@ export const InferencePipelineCard: React.FC<InferencePipeline> = (pipeline) =>
-              </EuiTitle>
+              <EuiFlexGroup alignItems="center">
+                <EuiFlexItem>
+                  <EuiTitle size="xs">
+                    <h4>{pipelineName ?? modelTitle}</h4>
+                  </EuiTitle>
+                </EuiFlexItem>
+                <EuiFlexItem grow={false} />
+              </EuiFlexGroup>
             </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <EuiPopover
-                button={actionButton}
-                isOpen={isPopOverOpen}
-                closePopover={() => setIsPopOverOpen(false)}
-              >
-                <EuiPopoverTitle paddingSize="m">
-                  {i18n.translate('xpack.enterpriseSearch.inferencePipelineCard.action.title', {
-                    defaultMessage: 'Actions',
-                  })}
-                </EuiPopoverTitle>
-                <EuiFlexGroup direction="column" gutterSize="none">
-                  <EuiFlexItem>
-                    <div>
-                      <EuiButtonEmpty
-                        data-telemetry-id={`entSearchContent-${ingestionMethod}-pipelines-inferencePipeline-stackManagement`}
-                        size="s"
-                        flush="both"
-                        iconType="eye"
-                        color="text"
-                        href={http.basePath.prepend(
-                          `/app/management/ingest/ingest_pipelines/?pipeline=${pipelineName}`
-                        )}
-                      >
-                        {i18n.translate(
-                          'xpack.enterpriseSearch.inferencePipelineCard.action.view',
-                          { defaultMessage: 'View in Stack Management' }
-                        )}
-                      </EuiButtonEmpty>
-                    </div>
-                  </EuiFlexItem>
-                  <EuiFlexItem>
-                    <div>
-                      <EuiButtonEmpty
-                        data-telemetry-id={`entSearchContent-${ingestionMethod}-pipelines-inferencePipeline-detachPipeline`}
-                        size="s"
-                        flush="both"
-                        iconType="unlink"
-                        color="text"
-                        onClick={() => {
-                          detachMlPipeline({ indexName, pipelineName });
-                          setIsPopOverOpen(false);
-                        }}
-                      >
-                        {i18n.translate(
-                          'xpack.enterpriseSearch.inferencePipelineCard.action.detach',
-                          { defaultMessage: 'Detach pipeline' }
-                        )}
-                      </EuiButtonEmpty>
-                    </div>
+            {modelTitle && (
+              <EuiFlexItem>
+                <EuiFlexGroup gutterSize="s">
+                  <EuiFlexItem grow={false}>
+                    <EuiTextColor color="subdued">{modelId}</EuiTextColor>
```

Comment:
In which cases can `modelTitle` be falsy? Does it make sense to hide `modelId` if that happens?

I'm guessing this is related to model redaction, which is an unusual but possible case:
- Model M is only sync'ed with Kibana space S1
- We create pipeline P while in Kibana space S1 and select model M
- We switch to Kibana space S2 and select existing pipeline P for an index. The pipeline is selectable (pipelines are not specific to spaces) and it works if executed, but due to security reasons the model ID is not displayed.

Reply:
What would we want to display instead?

Reply:
Ideally a message like "This model is not available in this Kibana space". This is not in scope for this PR, so let's go with the current implementation and perhaps please add an item to the list of tech debt.